core/core_test.go

```diff
@@ -177,100 +179,150 @@ func TestOrderSync(t *testing.T) {
 		t.Skip("Serial tests (tests which cannot run in parallel) are disabled. You can enable them with the --serial flag")
 	}
 
-	teardownSubTest := setupSubTest(t)
-	defer teardownSubTest(t)
-
-	// Set up two Mesh nodes. originalNode starts with some orders. newNode enters
-	// the network without any orders.
-	ctx, cancel := context.WithTimeout(context.Background(), 60*time.Second)
-	defer cancel()
-	wg := &sync.WaitGroup{}
-
-	perPage := 10
-	pConfig := privateConfig{
-		paginationSubprotocolPerPage: perPage,
+	testCases := []ordersyncTestCase{
+		// NOTE(albrow): The test case below is just a sanity check to make sure
```

Comment:
We can't just uncomment the test case because it is designed to fail.  This was only here during development to make sure that `TestOrderSync` itself was working correctly (i.e. that it was testing what it was supposed to test).

We could potentially restructure this by writing a test that passes if there are no messages received after a certain amount of time. Honestly though I don't think it's worth it. There is no situation where we would intentionally start `core.App` with no ordersync subprotocols enabled. Tests which try to make sure an event _does not_ happen are also pretty finicky in general and I try to avoid them.

Alternatively I would be okay with just removing this test case and the associated comment. What do you think?

Reply:
Let's just remove it and open an issue or write a `TODO` comment if you think that this failing test is useful to people who are going to work on `ordersync`.

Reply:
I just decided to remove it.